core/jazz_ui/src/app/pages/service-detail/service-detail.component.ts

```diff
@@ -45,7 +45,7 @@ export class ServiceDetailComponent implements OnInit {
 
   @Output() deleteServiceStatus: EventEmitter<boolean> = new EventEmitter<boolean>();
   @ViewChild('selectedTabComponent') selectedTabComponent;
-  isENVavailable: boolean = true;
+  isENVavailable: boolean = false;
```

Comment:
Can we name it as isEnvAvailable so it is more readable? Also, before adding new variables make sure we do NEED this additional variable? Let us use camelCase the variable names going forward.

Reply:
Updated the variable to camelCase. This variable was already available ,I just use it.

Reply:
LGTM